src/app/pages/private/LifeListPage/displayLife.tsx

```diff
@@ -0,0 +1,28 @@
+import { useListLivesQuery } from '../../../api';
+import NoItemResult from '../../../components/results/NoItemResult';
+
+const DisplayLife = () => {
+    const { data, loading } = useListLivesQuery();
+
+    if (loading) {
+        // not yet loaded
+        return null;
+    }
+
+    const badam = data?.lives || [];
+    console.log(badam)
+    if (badam.length === 0) {
+        return <NoItemResult />
+    }
+
+    const DisplayData = () => {
+        return (
+            <>
+            {badam.map((life) => <p>{life.id} {life.title} {life.fullName}</p>)}
+            </>
+        )
+    }
```

Comment:
You cannot create a component inside a component rendering ticks. If you do it should be memoized but here there's no reason to do such thing.